python/vineyard/cli.py

```diff
@@ -134,62 +184,121 @@ def as_object_id(object_id):
+        objects = client.list_objects(pattern=args.pattern, regex=args.regex, limit=args.limit)
         print(f"List of your vineyard objects:\n{objects}")
     except BaseException as exc:
         raise Exception(f'The following error was encountered while listing vineyard objects:\n{exc}') from exc
 
 
-def get(client, object_id):
+def query(client, args):
     """Utility to fetch a vineyard object based on object_id."""
     try:
-        value = client.get_object(as_object_id(object_id))
+        value = client.get_object(as_object_id(args.object_id))
         print(f"The vineyard object you requested:\n{value}")
     except BaseException as exc:
+        if args.exists:
+            print(f"The object with object_id({args.object_id}) doesn't exists")
         raise Exception(('The following error was encountered while fetching ' +
-                         f'the vineyard object({object_id}):\n{exc}')) from exc
-
-
-def delete_obj(client, object_id, force, deep):
+                         f'the vineyard object({args.object_id}):\n{exc}')) from exc
+
+    if args.exists:
+        print(f'The object with object_id({args.object_id}) exists')
+    if args.stdout:
+        sys.stdout.write(str(value) + '\n')
+    if args.output_file is not None:
+        with open(args.output_file, 'w') as output_file:
+            output_file.write(str(value))
+    if args.meta is not None:
+        if args.meta == 'simple':
+            print(f'Meta data of the object:\n{value.meta}')
+        elif args.meta == 'json':
+            json_meta = json.dumps(value.meta, indent=4)
+            print(f'Meta data of the object in JSON format:\n{json_meta}')
+    if args.metric is not None:
+        print(f'{args.metric}: {getattr(value, args.metric)}')
+
+
+def delete_obj(client, args):
     """Utility to delete a vineyard object based on object_id."""
     try:
-        client.delete(object_id=as_object_id(object_id), force=force, deep=deep)
-        print(f'The vineyard object({object_id}) was deleted successfully')
+        client.delete(object_id=as_object_id(args.object_id), force=args.force, deep=args.deep)
+        print(f'The vineyard object({args.object_id}) was deleted successfully')
     except BaseException as exc:
         raise Exception(('The following error was encountered while deleting ' +
-                         f'the vineyard object({object_id}):\n{exc}')) from exc
+                         f'the vineyard object({args.object_id}):\n{exc}')) from exc
 
 
-def status(client, properties):
+def status(client, args):
     """Utility to print the status of connected vineyard server."""
     stat = client.status
-    if properties is None:
+    if args.properties is None:
         print(stat)
     else:
         print('InstanceStatus:')
-        for prop in properties:
+        for prop in args.properties:
             print(f'    {prop}: {getattr(stat, prop)}')
 
 
+def put_obj(client, args):
```

Comment:
Naming: `put_obj` -> `put` or `put_object`.

Reply:
Renamed `put_obj` to `put_object`